src/api/models/base.py

```diff
@@ -55,21 +55,15 @@ def __str__(self):
 
     @property
     def full_name(self):
-        name = []
-        if self.first_name:
-            name.append(self.first_name)
-
-        if self.last_name:
-            name.append(self.last_name)
-
-        if len(name) > 0:
-            return ' '.join(name)
+        if self.first_name and self.last_name:
```

Comment:
I'm thinking this shouldn't be `and` but rather `or`, and then use the existing dev logic. So a person with only first or last name populated would still count.